tracer/src/Datadog.Trace/DiagnosticListeners/AspNetCoreDiagnosticObserver.cs

```diff
@@ -533,8 +533,11 @@ private static Span StartMvcCoreSpan(Tracer tracer, Span parentSpan, BeforeActio
             {
                 // If we're using endpoint routing or this is a pipeline re-execution,
                 // these will already be set correctly
-                parentTags.AspNetCoreRoute = aspNetRoute;
                 parentSpan.ResourceName = span.ResourceName;
+                if (parentTags != null)
```

Comment:
minor, but I don't believe `parentTags` _can_ be null here? `StartMvcCoreSpan` is only called with endpoint routing, and we always create those tags?